bootstrapers/showcase/ShowcaseThemeResolver.tsx

```diff
@@ -16,6 +16,8 @@ export const ShowcaseThemeResolver = (props: {
   path: string;
   templateId: number;
   data: GraphContent[];
+  footer:GraphContent;
+  header:GraphContent;
```

Comment:
```suggestion
  footer: GraphContent;
  header: GraphContent;
```
I think you are forgetting to format the file before saving :)

Reply:
Sorry